site/content/docs/user/kuberay.md

```diff
@@ -0,0 +1,278 @@
+---
+title: "Ray on GKE using DraNet"
+date: 2025-07-14T10:10:40Z
+---
+
+
+To get started, follow the instructions to create a [GKE cluster with DRA support and using DraNet](./gke-rdma.md)
+
+### Deploy RayCluster
+
+Install CRD and KubeRay operator
+
+```sh
+kubectl create -k "github.com/ray-project/kuberay/ray-operator/config/default?ref=v1.4.1"
+```
+
+We create one `ResourceClaimTemplate`, for the RDMA devices on the node, along with a `DeviceClass` for the RDMA device.
+
+```yaml
+apiVersion: resource.k8s.io/v1beta1
+kind: DeviceClass
+metadata:
+  name: dranet
+spec:
+  selectors:
+    - cel:
+        expression: device.driver == "dra.net"
+---
+apiVersion: resource.k8s.io/v1beta1
+kind: ResourceClaimTemplate
+metadata:
+  name: all-nic
+spec:
+  spec:
+    devices:
+      requests:
+      - name: nic
+        deviceClassName: dranet
+        count: 8
```

Comment:
Does using DRANet allow us to use < 8 NICs (or whatever amount less than total amount of GPUs on the node)?

Reply:
per example, you can define workers of 4 GPUs and add a claim to also attach the 4 associated NICs, that will allow you to have 4 workers in 2 A4 nodes with 8 GPUS